roles/webhooks/main.yml

```diff
@@ -0,0 +1,11 @@
+# Copy local webhooks into webhooks folder
+- name: copy webhook plugins
+  copy:
+    src: "{{ webhook_plugins_src_dir }}/{{ item }}"
```

Comment:
src: "{{ webhook_plugins_src_dir }}/{{ item }}.yml.j2"

Reply:
This probably wouldn't work because "{{ webhook_plugins_src_dir }}/{{ item }}" is a directory..